spec/apps/invalid-mount/invalid-mount.spec.js

```diff
@@ -43,8 +43,6 @@ describe(`invalid-mount app`, () => {
         return singleSpa
           .triggerAppChange()
           .then(() => {
-            // doesn't get unmounted because it's in a broken state.
-            expect(myApp.wasUnmounted()).toEqual(false);
```

Comment:
No longer true, as per our core team discussion last week.

Reply:
nit: may consider changing the `it` statement above to reflect this change as well